config_defaults_inc.php

```diff
@@ -2153,6 +2153,14 @@
  */
 $g_ldap_simulation_file_path = '';
 
+/**
+ * The LDAP fields of a user to be cached
+ * @global array $g_ldap_cache_fields
+**/
+$g_ldap_cache_fields = array(
+	'mail'
```

Comment:
Having `mail` field here doesn't seem to be needed.

Reply:
Yes, it was just weird to have an empty array

Reply:
https://www.mantisbt.org/bugs/view.php?id=26809